CD/AWS_ECS

```diff
@@ -182,6 +183,11 @@ pipeline
     }     	
     post
     {
+        always {
+            script {
+                def response = httpRequest ignoreSslErrors: true, url: "https://devops.app.geeks.ltd/api/build-completed/$JOB_ID"
```

Comment:
expected to send GET request, like:
`https://devops.app.geeks.ltd/api/build-completed/35b09bfa-330d-42b5-823a-5c6712eb1021`

Reply:
Please add Jenkins **HTTP Request** if you have not installed it before:
[HTTP Request](https://plugins.jenkins.io/http_request)